hummingbot/connector/exchange/bitmart/bitmart_order_book_tracker.py

```diff
@@ -71,21 +66,7 @@ async def _track_single_book(self, trading_pair: str):
                 else:
                     message = await message_queue.get()
 
-                if message.type is OrderBookMessageType.DIFF:
-                    bids, asks = bitmart_utils.convert_diff_message_to_order_book_row(message)
-                    order_book.apply_diffs(bids, asks, message.update_id)
-                    past_diffs_window.append(message)
-                    while len(past_diffs_window) > self.PAST_DIFF_WINDOW_SIZE:
-                        past_diffs_window.popleft()
-                    diff_messages_accepted += 1
-
-                    # Output some statistics periodically.
-                    now: float = time.time()
-                    if int(now / 60.0) > int(last_message_timestamp / 60.0):
-                        self.logger().debug(f"Processed {diff_messages_accepted} order book diffs for {trading_pair}.")
-                        diff_messages_accepted = 0
-                    last_message_timestamp = now
-                elif message.type is OrderBookMessageType.SNAPSHOT:
```

Comment:
As per the previous comment, this should still be required

Reply:
Same as above, there is no diff message.